packages/commons-ui-core/.lintstagedrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require("eslint-config-commons-ui/.lintstagedrc.js");
```

Comment:
Looks weird requiring lintstaged from eslint. Add content here as we do in other projects is much better me thinks.